react/nextJsQuestions.md

```diff
@@ -0,0 +1,37 @@
+# Next.js Interview Questions
+
+## General Questions:
+
+[What is Next.js, and how does it differ from traditional React applications?](https://nextjs.org/docs/getting-started/introduction)
+
+[Explain the concept of server-side rendering (SSR) in Next.js](https://nextjs.org/docs/basic-features/data-fetching)
```

Comment:
Replace to https://nextjs.org/docs/pages/building-your-application/data-fetching